### PR TITLE
Add galog presubmit checks

### PR DESCRIPTION
**Pull request by drewhli**
/cc @dorileo @ChaitanyaKulkarni28

### CODE REVIEW

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+presubmits:
+  GoogleCloudPlatform/galog:
+  - name: galog-presubmit-gocheck
+    cluster: gcp-guest
+    run_if_changed: '.*'
```
Comment (dorileo) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml:5]: please reduce the scope to `'\.go$'`

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    trigger: "(?m)^/gocheck$"
+    rerun_command: "/gocheck"
+    context: prow/presubmit/gocheck/galog
```
Comment (dorileo) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml:8]: I believe this should be `context: prow/presubmit/gocheck`

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    spec:
+      containers:
+      - image: gcr.io/gcp-guest/gocheck:latest
+        imagePullPolicy: Always
+        command:
+        - "/go/main.sh"
+        args: ["."]
+  - name: galog-presubmit-gotest
+    cluster: gcp-guest
+    run_if_changed: '.*'
```
Comment (dorileo) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml:19]: ditto

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    trigger: "(?m)^/gotest$"
+    rerun_command: "/gotest"
+    context: prow/presubmit/gotest/galog
```
Comment (dorileo) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml:22]: ditto

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    rerun_command: "/gocheck"
+    context: prow/presubmit/gocheck/galog
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/gcp-guest/gocheck:latest
+        imagePullPolicy: Always
+        command:
+        - "/go/main.sh"
+        args: ["."]
```
Comment (dorileo) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml:16]: is this arg required?

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/gcp-guest/gotest:latest
+        imagePullPolicy: Always
+        command:
+        - "/go/main.sh"
+  - name: galog-presubmit-gobuild
+    cluster: gcp-guest
+    run_if_changed: '.*'
```
Comment (dorileo) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml:32]: ditto

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    trigger: "(?m)^/gobuild$"
+    rerun_command: "/gobuild"
+    context: prow/presubmit/gobuild/galog
```
Comment (dorileo) [inline prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml:35]: ditto

**Review by dorileo — CHANGES_REQUESTED** [review]

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml (hunk 5 above)
Comment (drewhli) [reply prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml:16]: No, I forgot to remove it.

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml (hunk 6 above)
Comment (drewhli) [reply prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml]: Done.

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml (hunk 4 above)
Comment (drewhli) [reply prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml]: Done.

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml (hunk 3 above)
Comment (drewhli) [reply prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml]: Done.

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml (hunk 2 above)
Comment (drewhli) [reply prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml]: Done.

File: prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml (hunk 1 above)
Comment (drewhli) [reply prow/prowjobs/GoogleCloudPlatform/gcp-guest/galog.yaml]: Done.

**Pull request merged**